engine/rpcserver_test.go

```diff
@@ -811,12 +824,13 @@ func TestUpdateAccountInfo(t *testing.T) {
 	}
 
 	updateResponse, err := s.UpdateAccountInfo(context.Background(), &gctrpc.GetAccountInfoRequest{Exchange: fakePassExchange, AssetType: asset.Futures.String()})
-	if err != nil {
-		t.Fatalf("TestGetAccountInfo: Failed to update account info: %s", err)
+	if !errors.Is(err, errAssetTypeDisabled) {
+		t.Errorf("expected %v, received %v", errAssetTypeUnset, err)
 	}
-
-	if getResponse.Accounts[0].Currencies[0].TotalValue == updateResponse.Accounts[0].Currencies[0].TotalValue {
-		t.Fatalf("TestGetAccountInfo: Unexpected value of the 'TotalValue'")
+	if err == nil {
```

Comment:
This is strange. because we are going to be erroring above on ln 826, it should be its own case.